# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../madwifi/eth-naming.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- madwifi/ath/if_ath.c.vanilla	2007-08-25 16:44:34.000000000 +0200
+++ madwifi/ath/if_ath.c	2007-08-25 16:44:41.000000000 +0200
@@ -951,7 +951,7 @@
 
 	if (autocreatemode != -1) {
 		rtnl_lock();
-		vap = ieee80211_create_vap(ic, "ath%d", dev,
+		vap = ieee80211_create_vap(ic, "eth%d", dev,
 				autocreatemode, 0);
 		rtnl_unlock();
 		if (vap == NULL)
--- madwifi-hal-0.9.30.13-r2351-20070519.orig/tools/80211debug.c	2006-12-09 03:20:08.000000000 +1000
+++ madwifi-hal-0.9.30.13-r2351-20070519/tools/80211debug.c	2007-06-03 20:49:02.000000000 +1000
@@ -38,7 +38,7 @@
 
 /*
  * 80211debug [-i interface] flags
- * (default interface is ath0).
+ * (default interface is eth1).
  */
 #include <sys/types.h>
 
@@ -183,7 +183,7 @@
 int
 main(int argc, char *argv[])
 {
-	const char *ifname = "ath0";
+	const char *ifname = "eth1";
 	const char *cp, *tp;
 	const char *sep;
 	int op, i;
--- madwifi-hal-0.9.30.13-r2351-20070519.orig/tools/80211stats.c	2007-03-06 20:59:28.000000000 +1000
+++ madwifi-hal-0.9.30.13-r2351-20070519/tools/80211stats.c	2007-06-03 20:49:02.000000000 +1000
@@ -252,7 +252,7 @@
 	struct iwreq iwr;
 	int allnodes = 0;
 
-	const char *ifname = "ath0";
+	const char *ifname = "eth1";
 
 	s = socket(AF_INET, SOCK_DGRAM, 0);
 	if (s < 0)
--- madwifi-hal-0.9.30.13-r2351-20070519.orig/tools/athdebug.c	2006-12-09 03:20:08.000000000 +1000
+++ madwifi-hal-0.9.30.13-r2351-20070519/tools/athdebug.c	2007-06-03 20:49:02.000000000 +1000
@@ -38,7 +38,7 @@
 
 /*
  * athdebug [-i interface] flags
- * (default interface is wifi0).
+ * (default interface is eth1).
  */
 #include <sys/types.h>
 #include <sys/file.h>
@@ -183,9 +183,9 @@
 main(int argc, char *argv[])
 {
 #ifdef __linux__
-	const char *ifname = "wifi0";
+	const char *ifname = "eth1";
 #else
-	const char *ifname = "ath0";
+	const char *ifname = "eth1";
 #endif
 	const char *cp, *tp;
 	const char *sep;
diff -urN madwifi-hal-0.9.30.13-r2351-20070519.orig/tools/athstats.c madwifi-hal-0.9.30.13-r2351-20070519/tools/athstats.c
--- madwifi-hal-0.9.30.13-r2351-20070519.orig/tools/athstats.c	2007-03-27 11:44:05.000000000 +1000
+++ madwifi-hal-0.9.30.13-r2351-20070519/tools/athstats.c	2007-06-03 20:49:02.000000000 +1000
@@ -40,7 +40,7 @@
  * Simple Atheros-specific tool to inspect and monitor network traffic
  * statistics.
  *	athstats [-i interface] [interval]
- * (default interface is wifi0).  If interval is specified a rolling output
+ * (default interface is eth1).  If interval is specified a rolling output
  * is displayed every interval seconds.
  */
 #include <sys/types.h>
@@ -230,9 +230,9 @@
 main(int argc, char *argv[])
 {
 #ifdef __linux__
-	const char *ifname = "wifi0";
+	const char *ifname = "eth1";
 #else
-	const char *ifname = "ath0";
+	const char *ifname = "eth1";
 #endif
 	int s;
 	struct ifreq ifr;
